week2-concurrency.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+	"sync"
+	"os"
+	"path/filepath"
+)
+
+var global_result = make(map[string]int)
+var numberOfJobs int = 3
+var numberOfBufferSizes int = 16
+
+func readAllFilePaths(pathFolder string) ([]string, error) {
+	var filePaths []string
+	err := filepath.Walk(pathFolder,
+		func(path string, info os.FileInfo, err error) error {
```

Comment:
why new line here? It's hard to read.

Reply:
i don't want who read my code be eyestrain because line too long :)
but look again, that really hard to read :v thanks